admin/templates/users/remove_spam_user.html

```diff
@@ -0,0 +1,17 @@
+<form class="well" method="post" action="{% url 'users:spam_disable' guid=guid %}">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">x</button>
+        <h3>Are you sure you want to delete this user and mark all their nodes as private? {{ guid }}</h3>
+    </div>
+    <div class="modal-body">
+        Parts of this action will be reversible after the fact.
```

Comment:
This is... cryptic. Perhaps add information about which parts of this action will be irreversible (marking nodes private?). Also not a blocker.